include/Core/Config.php

```diff
@@ -132,15 +132,33 @@ public static function set($family,$key,$value) {
 		$dbvalue = (is_array($value)?serialize($value):$value);
 		$dbvalue = (is_bool($dbvalue) ? intval($dbvalue) : $dbvalue);
 
-		$ret = q("INSERT INTO `config` ( `cat`, `k`, `v` ) VALUES ( '%s', '%s', '%s' )
-ON DUPLICATE KEY UPDATE `v` = '%s'",
+		// The "INSERT" command is very cost intense. It saves performance to do it this way.
```

Comment:
In general, I don't favor implicit behavior because it raises the entry bar for new collaborators, whether it's a public or private project. But it could be a viable solution if there indeed is a performance problem, which I don't feel there is right now, whether from a server load perspective or a user experience one.